scio-core/src/main/scala/com/spotify/scio/schemas/To.scala

```diff
@@ -139,27 +139,28 @@ object To {
 }
 
 object ToMacro {
-  import scala.reflect.macros.blackbox
+  import scala.reflect.macros._
   def safeImpl[I: c.WeakTypeTag, O: c.WeakTypeTag](
-    c: blackbox.Context
+    c: whitebox.Context
```

Comment:
Why does this need to be whitebox ?

Reply:
ah should be reverted now, I was just playing around with some stuff.